DROD/CharacterOptionsWidget.cpp

```diff
@@ -68,23 +68,27 @@ CCharacterOptionsDialog::CCharacterOptionsDialog(
 void CCharacterOptionsDialog::SetCharacter(
 	const CCharacter *pCharacter
 ){
-	WCHAR temp[8];
+	WCHAR temp[10];
```

Comment:
I'm expecting we need to address this overflow in the rpg codebase as well, correct?

Reply:
I don't know if RPG has processing sequence editing - this was introduced late and I do not remember ever porting it.

Reply:
Ah, I didn't look closely enough here.  Didn't realize this was specific in the character options dialog box.
I agree this isn't relevant to RPG.